pkg/data/values.go

```diff
@@ -1,5 +1,14 @@
+// Package data contains functions for working with unstructured values like interfaces or map[string]interface{}
```

Comment:
```suggestion
// Package data contains functions for working with unstructured values like []interface and map[string]interface{}.
// It allows reading and mutating them without having to convert them to structured values.
```

Reply:
I've made some adjustments to the doc comment. LMK if you want any further changes.